scripts/zones/Bibiki_Bay/mobs/Peerifool.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Bibiki Bay
+--  mob: Peerifool
+--  Quest: One Good Deed?
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.SLEEP, {chance = 100})
+end
+
+entity.onMobDeath = function(mob, player, isKiller)
+    local party = player:getParty()
+
+    for _, v in ipairs(party) do
```

Comment:
this whol block need to be moved to the quest lua under onMobDeath

Reply:
That's being fixed here: https://github.com/AirSkyBoat/AirSkyBoat/pull/397/files#diff-4dfac83c80926773842a66b202e55d2e72d5c3e6e43023e44018d2a134d2a6aa